edweiss-app/components/memento/DeleteDeckModalAction.tsx

```diff
@@ -0,0 +1,42 @@
+import ReactComponent from '@/constants/Component';
+import { BottomSheetModalMethods } from '@gorhom/bottom-sheet/lib/typescript/types';
+import { useState } from 'react';
+import TView from '../core/containers/TView';
+import ModalContainer from '../core/modal/ModalContainer';
+import TText from '../core/TText';
+import FancyButton from '../input/FancyButton';
+
+export const DeleteOptionModalDisplay: ReactComponent<{
```

Comment:
Great job on separating your components in different files: this is more readable than what I did (putting every modal component in the same file).

Reply:
xDD thank you. It took me some times to organize things but afterwards, it is more readable.